tasks/task03/words_stats.py

```diff
@@ -0,0 +1,30 @@
+"""
+You are given a list of sentences. Find all unique words along with:
+Number of word occurrences
+Number of the sentence where the word appeared for the first time
+
+Note: ignore word case during comparison, i.e. treat “Home”, “home” and “HOME” as the same words.
+"""
+from re import sub
+
+texts = [
+    "Hello, World!",
+    "The world is mine",
+    "Hello, how are you?"
+]
+
+ci_texts = list(map(lambda text: sub('([^A-z^\s])+', '', text).lower().split(), texts))
+accumulator = {}
+
+for words in ci_texts:
+    index = ci_texts.index(words)

```

Comment:
If you want to iterate over a list and get index and value you can use 
`for line_number, line in enumerate(ci_texts):`

Reply:
It would save one line 
```
for line_number, line in enumerate(ci_texts):
    for word in line:
```

Reply:
Reworked